sdk/identity/ci.yml

```diff
@@ -30,6 +30,7 @@ extends:
       LiveTestCtestRegex: azure-identity.
       LineCoverageTarget: 95
       BranchCoverageTarget: 56
+      UseFederatedAuth: true
```

Comment:
@gearama Is this necessary? It was my understanding that federated auth is on by default in the pipelines and you only need to set this to false when opting out

Reply:
That doesn't seem to be true, see similar change in ci.yml here:
https://github.com/Azure/azure-sdk-for-cpp/pull/5754/files#diff-126f4d961feeee67d4686c25c0682043833c8b596697eb2117f9da8735cbf38bR33

And @gearama's response to your question:
https://github.com/Azure/azure-sdk-for-cpp/pull/5754#discussion_r1668978160

> RickWinter: Is this necessary if default is true?
> gearama: yes